examples/mobilenet/tfckpt2npz.py

```diff
@@ -0,0 +1,177 @@
+            "gamma":
+            ckpt_reader.get_tensor(tf_scope + '/depthwise/BatchNorm/gamma'),
+            "moving_mean":
+            ckpt_reader.get_tensor(
+                tf_scope + '/depthwise/BatchNorm/moving_mean'),
+            "moving_variance":
+            ckpt_reader.get_tensor(
+                tf_scope + '/depthwise/BatchNorm/moving_variance')
+        }
+
+        # Project convolution parameters
+        project_params = {
+            "weights":
+            ckpt_reader.get_tensor(tf_scope + '/project/weights'),
+            "beta":
+            ckpt_reader.get_tensor(tf_scope + '/project/BatchNorm/beta'),
+            "gamma":
+            ckpt_reader.get_tensor(tf_scope + '/project/BatchNorm/gamma'),
+            "moving_mean":
+            ckpt_reader.get_tensor(
+                tf_scope + '/project/BatchNorm/moving_mean'),
+            "moving_variance":
+            ckpt_reader.get_tensor(
+                tf_scope + '/project/BatchNorm/moving_variance'),
+        }
+        load_expanded_conv(
+            expanded_conv,
+            expand_params=expand_params,
+            depthwise_params=depthwise_params,
+            project_params=project_params,
+        )
+    # Similarly loading the vanilla convolutions.
+    # Initial convolution
+    init_conv_with_tf_weights(
+        model.conv.conv,
+        weights=ckpt_reader.get_tensor('MobilenetV2/Conv/weights'))
+    init_bn_with_tf_params(
+        model.conv.bn,
+        beta=ckpt_reader.get_tensor('MobilenetV2/Conv/BatchNorm/beta'),
+        gamma=ckpt_reader.get_tensor('MobilenetV2/Conv/BatchNorm/gamma'),
+        moving_mean=ckpt_reader.get_tensor(
+            'MobilenetV2/Conv/BatchNorm/moving_mean'),
+        moving_variance=ckpt_reader.get_tensor(
+            'MobilenetV2/Conv/BatchNorm/moving_variance'))
+    # Final convolution before dropout (conv1)
+    init_conv_with_tf_weights(
+        model.conv1.conv,
+        weights=ckpt_reader.get_tensor('MobilenetV2/Conv_1/weights'))
+    init_bn_with_tf_params(
+        model.conv1.bn,
+        beta=ckpt_reader.get_tensor('MobilenetV2/Conv_1/BatchNorm/beta'),
+        gamma=ckpt_reader.get_tensor('MobilenetV2/Conv_1/BatchNorm/gamma'),
+        moving_mean=ckpt_reader.get_tensor(
+            'MobilenetV2/Conv_1/BatchNorm/moving_mean'),
+        moving_variance=ckpt_reader.get_tensor(
+            'MobilenetV2/Conv_1/BatchNorm/moving_variance'))
+    # Logits convolution
+    init_conv_with_tf_weights(
+        model.logits_conv,
+        weights=ckpt_reader.get_tensor(
+            'MobilenetV2/Logits/Conv2d_1c_1x1/weights'),
+        bias=ckpt_reader.get_tensor('MobilenetV2/Logits/Conv2d_1c_1x1/biases'))
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        'model_name', choices=('mobilenetv2', ), default='mobilenetv2')
+    parser.add_argument('pretrained_model')
+    parser.add_argument('--n_class', type=int, default=1001)
```

Comment:
please use `--n-class`, no underscore in argument

Reply:
please format other arguments, too

Reply:
OK